### PR TITLE
Checkpoint -> JKSV

### PR DESCRIPTION
**Pull request by el-emmo**

### CODE REVIEW

File: docs/about.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ This guide was written by staff members of the [Nintendo Homebrew Discord Server
     - **switchbrew** for [nx-hbloader](https://github.com/switchbrew/nx-hbloader) and [nx-hbmenu](https://github.com/switchbrew/nx-hbmenu).
     - **nwert** and **CTCaer** for [Hekate](https://github.com/CTCaer/hekate).
     - **WerWolv** for [EdiZon](https://github.com/WerWolv/EdiZon/releases).
+	- **J-D-K** for [JKSV](https://github.com/J-D-K/JKSV).
```
Comment (JeffVi) [inline docs/about.md:22]: Not sure about the indentation here

File: docs/extras/save_management.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+### JKSV
+Please read JKSV's [homepage](https://github.com/J-D-K/JKSV) for information on how to use it.
 
 !!! tip ""
-	![ExampleSwitchTheme](../extras/img/save_edizon.jpg)
+	![ExampleSwitchTheme](../extras/img/save_jksv.jpg)
 
 ### Checkpoint
-Please read Checkpoint's [usage page](https://github.com/FlagBrew/Checkpoint#usage) for information on how to use it.
+Please read Checkpoint's [homepage](https://github.com/SciresM/Checkpoint#usage) for information on how to use it.
```
Comment (JeffVi) [inline docs/extras/save_management.md:16]: Probably want to change the link to "https://github.com/SciresM/Checkpoint" to match with the "homepage" name

**Review by JeffVi — CHANGES_REQUESTED** [review]
Just being annoying
Otherwise looks good

File: docs/extras/save_management.md [hunk 3]
```diff
@@ -1,19 +1,19 @@
 # Save Management
 
-For save management,  [Edizon](https://github.com/WerWolv/EdiZon/releases) is recommended. It can be used to back up and restore game saves to your SD card. It also has the ability to edit saves for extra items etc. It also has support for Atmosphere's cheat engine, providing an easy way to download new cheats, as well as toggle them on or off.
+For save management, [JKSV](https://github.com/J-D-K/JKSV) is recommended. It can be used to back up and restore game saves to your SD card. 
 
-[Checkpoint](https://github.com/flagbrew/checkpoint/releases) is also a save manager. It can be used to back up and restore game saves to your SD card. It also has the ability to share save data over FTP and WiFi.
+[Checkpoint](https://github.com/SciresM/Checkpoint/releases/tag/ams-19-compat) is also a save manager. It can be used to back up and restore game saves to your SD card. It also has the ability to share save data over FTP and WiFi.
```
Comment (suchmememanyskill) [inline docs/extras/save_management.md:5]: I do not think we should just wildly link scires' release on our docs page. Maybe mention that the current release is broken, and that there's a semi-fixed build [here]?

**Review by suchmememanyskill — APPROVED** [review]

**Pull request merged**